man/git-reauthor.md

```diff
@@ -7,25 +7,26 @@ git-reauthor(1) -- Rewrite history to change author's identity
 
 ## DESCRIPTION
 
-Rewrite history to change author's identity.
+Lets you replace the author and/or committer identities in commits and tags.
 
-Goes through all existing commits and tags to selectively modify the identities:  
-- modify name and/or email  
-- affect author and/or committer identities  
-- specify the new name/email as parameters or use the ones from the Git config  
-- change all objects or only replace those from a specific identity (defined by its email)
+The command goes through all existing commits and tags in the history to selectively modify the identities present in those objects. All the other informations are preserved.  
```

Comment:
> Commits not matching the query are left untouched.

Actually it's a bit different than this. A commit which is not directly concerned by a change of identity itself will still have its hash changed if its parent commit was modified.

I've changed the description to better explained what I meant there: "All the other informations such as dates, messages,.. are preserved"